pkg/execution/state/redis_state/queue_processor.go

```diff
@@ -500,7 +491,7 @@ func (q *queue) scan(ctx context.Context) error {
 	}
 
 	processAccount := false
-	if q.runMode.Account && (!q.runMode.Partition || rand.Intn(2) == 1) {
+	if q.runMode.Account && (!q.runMode.Partition || rand.Intn(100) <= q.runMode.AccountWeight) {
```

Comment:
We made the ratio of scanning accounts vs partitions configurable. Setting an AccountWeight of 85 means we scan accounts 85% of each scan and partitions 15%.